File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.js
function solution(my_string) {
    let answer = "";
    let arr = my_string.split("");
     // my_string을 배열로 변환한다
    for (let i = arr.length - 1; i >= 0; i--) {
        // i가 배열의 뒷쪽부터 앞으로 넣을 수 있게 for문을 선언한다	
        answer += arr[i];
        // answer에 배열의 뒷 부분. 즉, 문자열을 반대로 넣어준다
    }
    return answer;
}

File: 프로그래머스/unrated/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String answer = "";
        
        String front = my_string.substring(0, s);
		answer += front;
        // front에는 s이전의 문자열을 잘라서 넣는다
        
		String middle = my_string.substring(s, e + 1);
		for(int i = middle.length() - 1; i >= 0; i--) {
			answer += middle.charAt(i);
		} // middle에는 s와 e 사이의 문자열을 잘라 반복문을 이용해 반대로 넣어준다
        
		String back = my_string.substring(e + 1);
		answer += back;
        // back에는 나머지 문자열을 잘라서 넣는다
        
        return answer;
    }
}